src/vanilla/store2.ts

```diff
@@ -282,9 +256,7 @@ type PrdStore = {
   ) => Result
   sub: (atom: AnyAtom, listener: () => void) => () => void
 }
-type Store =
-  | (PrdStore & Partial<DevStoreRev2>)
-  | (PrdStore & DevStoreRev2 & DevStoreRev3)
+type Store = PrdStore | (PrdStore & DevStoreRev4)
```

Comment:
Any chance we could export `PrdStore & DevStoreRev4`?

Reply:
I can do that.